imports/ui/report-explorer/report-explorer.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react'
+import { Meteor } from 'meteor/meteor'
+import { connect } from 'react-redux'
+import { createContainer } from 'meteor/react-meteor-data'
+import PropTypes from 'prop-types'
+import RootAppBar from '../components/root-app-bar'
+import memoizeOne from 'memoize-one'
+import Reports, { collectionName } from '../../api/reports'
+import Preloader from '../preloader/preloader'
+import { setDrawerState } from '../general-actions'
+import Units, { collectionName as unitsCollName } from '../../api/units'
+import { NoItemMsg } from '../explorer-components/no-item-msg'
+import { FilterRow } from '../explorer-components/filter-row'
+import { ItemRow } from '../explorer-components/item-row'
+import { UnitBtn } from '../explorer-components/unit-btn'
+
+class ReportExplorer extends Component {
+  constructor () {
+    super(...arguments)
+    this.state = {
+      filterStatus: true,
+      myInvolvement: false,
+      expandedUnits: []
+    }
+  }
+
+  handleStatusClicked = (value) => {
+    this.setState({ filterStatus: value })
+  }
+
+  handleAssignedClicked = () => {
```

Comment:
This should not be named "handleAssignedClicked", as it represents the "owned by me" status